suzieq/engines/pandas/mroutes.py

```diff
@@ -0,0 +1,38 @@
+from typing import List
+from ipaddress import ip_address, ip_network
+from collections import defaultdict
+
+import numpy as np
+import pandas as pd
+
+from suzieq.engines.pandas.engineobj import SqPandasEngine
+
+
+class MroutesObj(SqPandasEngine):
+    '''Backend class to handle manipulating mroutes table with pandas'''
+
+    @staticmethod
+    def table_name():
+        '''Table name'''
+        return 'mroutes'
+
```

Comment:
If possible, I would ask you to code also the summarize function. The default implementation doesn't give relevant info about the table.